src/YesSql.Core/Sql/SqlBuilder.cs

```diff
@@ -79,30 +79,40 @@ public virtual void InnerJoin(string table, string onTable, string onColumn, str
             // Don't prefix if alias is used
```

Comment:
As noted here I removed prefixes for alias so either I revert this change or I keep it and we create a migration in Orchard Core.